scripts/run_server.sh

```diff
@@ -3,5 +3,21 @@
 set -e
 set -o pipefail
 
-poetry install
+# We need an older version of setuptools (and pip) to install gym 0.21. Poetry
```

Comment:
This should not be committed. While it speeds up local development if you need both scripts, running the server alone does not require gym and users should not be required to install it.

Long-term we should update to `gymnasium` to avoid this workaround.

Reply:
what would be a feasible short term solution? how about removing all changes in this file from this pull request, then adding run_server.sh to the .gitignore?

Reply:
You can just keep the local change without committing it. So remove all changes from this PR, but you shouldn't add it to the gitignore since changes to that file should not generally be ignored -- just this particular one. I think we can get to #21 pretty soon, so this should not be an issue much longer.